pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/defaultvalues/RgpAutomaticProbandDataCreator.java

```diff
@@ -37,8 +40,28 @@ protected boolean setDefaultData() {
                     fieldSettings.getColumnsWithDefaultOptionsFilteredByElasticExportWorkflow(fieldSettingsDtosByOptionAndInstanceId);
             String participantId = StringUtils.isNotBlank(esProfile.getLegacyAltPid()) ? esProfile.getLegacyAltPid() : esProfile.getGuid();
             ParticipantData participantData = new ParticipantData(participantDataDao);
-            Optional<BookmarkDto> maybeFamilyIdOfBookmark = bookmarkDao.getBookmarkByInstance(RGP_FAMILY_ID);
-            Map<String, String> probandDataMap = extractProbandDefaultDataFromParticipantProfile(maybeFamilyIdOfBookmark);
+
+            Optional<BookmarkDto> familyIdOfBookmark = bookmarkDao.getBookmarkByInstance(RGP_FAMILY_ID);
+            if (familyIdOfBookmark.isEmpty()) {
+                // nothing in the call stack will handle a throw properly, so log the error and use the return value
+                // to unwind
+                logger.error("RGP family ID not found in Bookmark table");
```

Comment:
Is error the right log level for this (WARN was what I was thinking) and would we benefit from logging additional information (participantId, requesting user, etc)?

Reply:
I think this is an error since we cannot proceed with the operation at this point. It replaces the former line 72 in this file, which currently throws but it only results in a warning. In any case, restructuring so this makes more sense.